boot/zephyr/keys.c

```diff
@@ -61,14 +61,14 @@ const unsigned char root_pub_der[] = {
     0x8b, 0x68, 0x34, 0xcc, 0x3a, 0x6a, 0xfc, 0x53,
     0x8e, 0xfa, 0xc1, };
 const unsigned int root_pub_der_len = 91;
-#else
-#error "No public key available for given signing algorithm."
```

Comment:
Why get rid of this error? Link will fail without it.

BTW, there is some desire to have at least the Zephyr make generate this key from a .pem file.